atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -160,6 +373,8 @@ def __init__(self, tasker, workflow, config_path, basename='config.yaml'):
         super(ReactorConfigPlugin, self).__init__(tasker, workflow)
         self.config_path = config_path
         self.basename = basename
+        self.reactor_config_map = os.environ.get('REACTOR_CONFIG', None)
```

Comment:
Could make `config_path` default to `None` now?

Reply:
no, as of now we are keeping still current functionality, and only providing methods to access data from REACTOR_CONFIG

Reply:
ahh ok, I understand now what you mean, will do that